### PR TITLE
Add README.md for segment trees

### PR DESCRIPTION
**Pull request by jedian**
Signed-off-by: Jedian jmb15@c3sl.ufpr.br

### CODE REVIEW

File: SegmentTrees/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+###Segment Trees
+
+A segment tree is a flexible data-structure built upon an array that stores any kind of information of the subsegments(intervals) of the base array.
+These information may be, for example, the sum of the elements in the interval or the biggest/smallest value in the interval.
```
Comment (erilyth) [inline SegmentTrees/README.md:4]: *This

File: SegmentTrees/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+This structure is basically maintained through three operations:
+
+* Build tree;
```
Comment (erilyth) [inline SegmentTrees/README.md]: Could you bold this

File: SegmentTrees/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  Given `N` the size of the array, recursively build the binary tree based on the array. Complexity `O(N log(N))`.
+* Query interval;
```
Comment (erilyth) [inline SegmentTrees/README.md]: Bold this as well

File: SegmentTrees/README.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  Given `N` the size of the array, `A` the left frontier of the interval, `B` the right frontier of the interval, recursively gets and returns the info stored on the tree for the asked(solicitado) interval. Complexity `O(log(N))`.
+* Update tree;
```
Comment (erilyth) [inline SegmentTrees/README.md]: And this too

**Review by erilyth — COMMENTED** [review]
Thanks! Could you please make the changes I suggested and then I'll merge.

**Comment by jedian** [comment]
@erilyth If that was all, done! :)

**Comment by erilyth** [comment]
@Jedian Thanks! :)

**Pull request merged**